gpii/node_modules/testing/src/Fixtures.js

```diff
@@ -308,13 +308,13 @@ fluid.defaults("gpii.test.checkSequence", {
 fluid.defaults("gpii.test.snapshotSequenceGrade", {
     gradeNames: ["gpii.test.couchSequenceGrade"],
     sequenceElements: {
-        login: {
-            gradeNames: "gpii.test.loginSequence",
-            priority: "after:startServer"
-        },
         snapshot: {
             gradeNames: "gpii.test.snapshotSequence",
-            priority: "after:login"
+            priority: "after:startServer"
```

Comment:
This seems a lot more likely to be the real solution to the problem.

Reply:
Looks like we picked up 1-2 other tests with just this fix.  Thanks again, @cindyli.